jupyterlab_git/git.py

```diff
@@ -567,12 +567,13 @@ async def log(self, path, history_count=10, follow_path=None):
 
     async def detailed_log(self, selected_hash, path):
         """
-        Execute git log -1 --numstat --oneline -z command (used to get
+        Execute git log -m -1 --numstat --oneline -z command (used to get
```

Comment:
So I guess the trick is in the various options for [`--diff-merges`](https://git-scm.com/docs/git-log#_diff_formatting), isn't it?

I'm looking forward to hearing what you learn by trying different options.